### PR TITLE
Keeps an out of bounds array reference from happening and updates Example pods

### LINKED ISSUE #21: Index of out range while configuring custom font sizes
My code:
`let parser = LTMarkdownParser()
parser.listAttributes.append([NSFontAttributeName: UIFont.systemFontOfSize(20.0)])
let attString = parser.attributedStringFromMarkdown(model.prayerRequests)!`

The code crashes at line 65 while trying to access what seems to be attributesArray at [-1]

### PR DESCRIPTION
**Pull request by SerenadeX**
Fixes #21 

Biggest problem was with the level not being checked that it was at least 0.

### CODE REVIEW

File: Example/Tests/LTMarkdownParserTests.swift [hunk 1]
```diff
@@ -598,4 +598,12 @@ class LTMarkdownParserTests: XCTestCase {
...
         XCTAssertEqual(attributedString?.string, "1.\u{00A0}Hi\n2.\u{00A0}Hi\n3.\u{00A0}Hi")
     }
+    
+    func testOutOfBoundsError() {
+        let parser = LTMarkdownParser()
+        parser.listAttributes.append([NSFontAttributeName: UIFont.systemFontOfSize(20.0)])
+        let markdown = "1. Hello\n2. hello\n\n+ hello\n+ hello"
+        let attString = parser.attributedStringFromMarkdown(markdown)
+        XCTAssertEqual(attString?.markdownString(), markdown)
+    }
```
Comment (brandenr) [inline Example/Tests/LTMarkdownParserTests.swift:608]: There should be an empty line above the final }

File: Example/Tests/LTMarkdownParserTests.swift [hunk 2]
```diff
@@ -598,4 +598,12 @@ class LTMarkdownParserTests: XCTestCase {
         let attributedString = standardParser.attributedStringFromMarkdown(markdownString)
         XCTAssertEqual(attributedString?.string, "1.\u{00A0}Hi\n2.\u{00A0}Hi\n3.\u{00A0}Hi")
     }
+    
+    func testOutOfBoundsError() {
+        let parser = LTMarkdownParser()
+        parser.listAttributes.append([NSFontAttributeName: UIFont.systemFontOfSize(20.0)])
```
Comment (brandenr) [inline Example/Tests/LTMarkdownParserTests.swift]: .0 Isn't needed

**Comment by brandenr** [comment]
.

File: Example/Pods/RichTextVC-iOS/src/Classes/RichTextViewController.swift [hunk 3]
```diff
@@ -473,108 +473,127 @@ public class RichTextViewController: UIViewController {
...
-
+        
         textView.attributedText = attributedText
     }
-
+    
+    private func removeFormattingFromListLeadsInRange(range: NSRange) {
+        guard let listHeadRegex = try? NSRegularExpression(pattern: "^(([0-9]+\\.\\u00A0)|(\\u2022\\u00A0)).*$", options: .AnchorsMatchLines),
+            regularFont = regularFont where
+            range.length > 0
```
Comment (stephanheilner) [inline Example/Pods/RichTextVC-iOS/src/Classes/RichTextViewController.swift:501]: Its a little weird that this is on the next line, not a deal breaker, but makes it harder to see the `where` condition.

Comment (brandenr) [inline Example/Pods/RichTextVC-iOS/src/Classes/RichTextViewController.swift:501]: I very much agree, same for the else, but this is in a pod update, not changed in this PR

Comment (SerenadeX) [inline Example/Pods/RichTextVC-iOS/src/Classes/RichTextViewController.swift:501]: I can make the adjustment in RichTextVC-iOS later.

**Comment by brandenr** [comment]
👍

**Pull request merged**